code/game/gamemodes/changeling/powers/strained_muscles.dm

```diff
@@ -13,20 +13,29 @@
 	var/enabled = 0 //Whether or not you are a hedgehog
 
 /datum/action/changeling/strained_muscles/sting_action(var/mob/living/carbon/user)
-	enabled = !enabled
-	if(enabled)
-		to_chat(user, "<span class='notice'>Our muscles tense and strengthen.</span>")
+	if(!enabled)
+		if(stacks > 0)
+			to_chat(user, "<span class='notice'>Our muscles are too tired to strengthen.</span>")
+		else if(user.dna.species.speed_mod < 0)
+			to_chat(user, "<span class='notice'>We are moving as fast as we can, we can not go faster.</span>")
+		else
+			to_chat(user, "<span class='notice'>Our muscles tense and strengthen.</span>")
+			enabled = !enabled
```

Comment:
```suggestion
			enabled = TRUE
```
There's only one possible value of `enabled` you can have getting to this line, but it's checked quite a bit above. Let's make it explicit.